include/mbedtls/platform.h

```diff
@@ -46,17 +46,32 @@ extern "C" {
  * \{
  */
 
+/* This macro tells if the current compiler is Visual Studio, before the 2015
+ * edition. It is an important piece of information due to the standard
+ * compliance issues breaking consistency across the supported platforms.
+ */
+#if defined(_MSC_VER) && _MSC_VER < 1900
+#define MSVC_LESS_THAN_14
```

Comment:
Throughout, I don't find `MSVC_LESS_THAN_14` very readable, and prefer `#if defined(_MSC_VER) && _MSC_VER < 1900` which is more readable and obvious.

Reply:
Also, since this name is defined in a public header, it should be namespaced.